webpack/assets/javascripts/foreman_navigation.js

```diff
@@ -0,0 +1,46 @@
+import $ from 'jquery';
+
+$(document).on('page:fetch', () => {
```

Comment:
please change this to a function and call it from the current event on page:fetch in application.js - it would make it much easier to find in the future.